### ISSUE TITLE
Changeset Api: HTTP 401 Unauthorized

### ISSUE THREAD

**Issue by eggix** [JOSM/changeset-viewer#8]
Requests to `https://osmcha.mapbox.com/api/v1/changesets/` return
```
HTTP 401 Unauthorized
Allow: GET, HEAD, OPTIONS
Content-Type: application/json
Vary: Accept
WWW-Authenticate: Token
{
    "detail": "Authentication credentials were not provided."
}
```
when using `Get Changeset in the area`, which renders the plugin unusable.

**Comment by Hb-** [comment]
Same issue here. What do?

**Comment by tsmock** [comment]
Fixed in 927029690d3bf9e4d01830106282aac535d5276f. I'll try to do a release next week.
